data/examples/rasa/demo-rasa.md

```diff
@@ -49,10 +49,15 @@
 - [central](location) [indian](cuisine) restaurant
 
 ## synonym:chinese
-- Chines
-- chines
-- Chinese
++ Chines
+* Chinese
 
 ## synonym:vegetarian
 - vegg
 - veggie
+
+## regex:zipcode
+- [0-9]{5}
```

Comment:
This suggest you can list multiple expressions for the same feature, does that work?

Reply:
I am not sure what the intended behavior of the regex featurizer is! If there are multiple patterns defined the md reader generates a json equivalent to:
```json
[{
"name": "zipcode",
"pattern": "pattern1"
},
{
"name": "zipcode",
"pattern": "pattern2"
}]
The regex featurizer doesn't care about the names and just adds the patterns as features it seems (https://github.com/RasaHQ/rasa_nlu/blob/master/rasa_nlu/featurizers/regex_featurizer.py#L81) . So I figured also multiple patterns per name should be fine, but I am not sure what's ultimately intended, since the json format doesn't really incentivize multiple patterns per name

Reply:
👍